src/cache.cpp

```diff
@@ -376,10 +376,17 @@ static const schema_patches schemaPatches{
 		{
 			"CREATE TABLE metadata ( "
 			" db_schema_version_major INTEGER NOT NULL, "
-			" db_schema_version_minor INTEGER NOT NULL );"
+			" db_schema_version_minor INTEGER NOT NULL );",
 
 			"INSERT INTO metadata VALUES ( 2, 11 );"
 		}
+	},
+	{	{2, 21},
```

Comment:
Makes sense.
I thought that wouldn't work because the version is being checked:
https://github.com/newsboat/newsboat/blob/7efb55f908d81b971c2bdfa2c37fd9c0e16aa8e5/src/cache.cpp#L393-L398
Turns out only the major version is checked so specifying 2.22 indeed works fine.